pkgs/applications/networking/remote/remmina/default.nix

```diff
@@ -23,7 +23,8 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-oEgpav4oQ9Sld9PY4TsutS5xEnhQgOHnpQhDesRFTeQ=";
   };
 
-  nativeBuildInputs = [ cmake ninja pkg-config wrapGAppsHook ];
```

Comment:
Clicking on `result/Applications/Remmina.app` in Finder doesn't do anything for me. `remmina-file-wrapper` seems to be looking for `remmina` in `$PATH` instead of the store location

```
> nix-build . -A remmina
/nix/store/vz6sx6lphzr1iklzxg4i6d89rq9f5gzn-remmina-1.4.31
> result/Applications/Remmina.app/Contents/MacOS/Remmina
/nix/store/vz6sx6lphzr1iklzxg4i6d89rq9f5gzn-remmina-1.4.31/bin/.remmina-file-wrapper-wrapped: line 52: remmina: command not found
```

Reply:
I was previously running into this `remmina-file-wrapper` issue previously but I didn't debug it at the time and it stopped happening as I had since installed `remmina` into `PATH`. I've added a patch that fixes this issue and submitted it upstream:

https://gitlab.com/Remmina/Remmina/-/merge_requests/2525